examples/gpio_sleep.rs

```diff
@@ -27,16 +27,27 @@ fn main() -> ! {
     let     swm    = p.SWM.split();
     let mut syscon = p.SYSCON.split();
     let mut wkt    = p.WKT.enable(&mut syscon.handle);
+    #[cfg(feature = "82x")]
+    let     gpio   = p.GPIO; // GPIO is initialized by default on LPC82x.
```

Comment:
I wonder if it's a good idea to disable GPIO on LPC82x in `Peripherals::take`. That's a bit of extra code and extra time for most applications, but it would make applications more portable. I don't know, and it has nothing to do with this pull request anyway. Just something that came to mind.